apps/website-25/src/components/homepage/CourseSection.test.tsx

```diff
@@ -34,7 +35,7 @@ describe('CourseSection', () => {
     // Verify GA event was sent with correct parameters
     expect(sendGAEvent).toHaveBeenCalledWith('event', 'course_card_click', {
       course_title: 'The Future of AI Course',
-      course_url: 'https://course.bluedot.org/future-of-ai/',
+      course_url: constants.COURSES[0]?.url,
```

Comment:
consider: If changing this I might also change the course_title and getByText arguments to use `COURSES[0]` - otherwise changing the content of the courses array will make things sad (tbf it probably already will)